samples/place-nearby-search/index.html

```diff
@@ -13,8 +13,19 @@
     <script type="module" src="./index.js"></script>
   </head>
   <body>
-    <div id="map"></div>
-
+      <gmp-map center="45.438646,12.327573" zoom="14" map-id="DEMO_MAP_ID">
+        <gmp-advanced-marker></gmp-advanced-marker>
+        <div id="controls" slot="control-inline-start-block-start">
+        <select name="types" class="type-select">
+          <option value="" selected disabled>Select a place type</option>
```

Comment:
should we fault to a real value and auto-trigger that on page load?
that way the demo doesn't look empty on init?